omod/src/test/java/org/openmrs/module/reportingrest/web/resource/EvaluatedCohortResourceTest.java

```diff
@@ -164,7 +164,7 @@ public void testEvaluatingSerializedCohortDefintionWithParams() throws Exception
 	private void assertCohortMembers(Object evaluated, String json, String[] expectedUuids) throws Exception {
 		assertEquals(expectedUuids.length, ((List) path(evaluated, "members")).size());
 		for (String expected : expectedUuids) {
```

Comment:
Was this intentional?

Reply:
Yes, due to change in dependency version, there is some change in the internals that cause this.. so had to change that. :)

Reply:
Does this mean the resource was changed from `patient` to `person`? Could you point me to where that was done?

Reply:
I'm not really sure why the change is happening. The problem is that the json built is having /person instead of /patient (which was the old case), so had to make the corresponding change in test also.
<img width="1330" alt="Screenshot 2021-06-22 at 12 33 06 PM" src="https://user-images.githubusercontent.com/37359019/122879032-0afe0580-d356-11eb-80d6-6dc32d7e9408.png">